governing-documents/SAMPLE-MAINTAINERS.md

```diff
@@ -103,7 +103,7 @@ The process to move a maintainer from active to emeritus status is comparable to
 resignation, the Pull Request can be merged following a maintainer PR approval. If the removal is for any other reason, the following steps **SHOULD** be followed:
 
 - A PR is created to update this file to move the maintainer to the list of emeritus maintainers.
-- The PR is authored by, or has a comment supporting the proposal from, an existing maintainer or Hyperledger GitHub organization administrator.
+- The PR is authored by, or has a comment supporting the proposal from, an existing maintainer or LFDT GitHub organization administrator (i.e., TAC).
```

Comment:
Same question as above: LFDT TAC or the project's TSC?

Reply:
Here I'd probably say TSC, but I would also be partial to mentioning both of them as an either or kinda thing.

Reply:
Personally, I think the project's TSC is enough (following the push-to-the-edge principle you mentioned above.)